### PR TITLE
Miscellaneous bug fixes and improvements

### PR DESCRIPTION
**Pull request by pranavxc**
## Change Summary

- [x] Create a project 
    - [x] Focus on the input field
    - [x] Generate default project name
- [x] Invite user modal close on escape key press ( after inviting a user ) '
- [x] Add new row Alt R : auto focus (edit mode) on first cell in newly added row
- [x] Right align cell for numerical fields (number, decimal, currency, duration, percent). Expanded form, should be left aligned
- [x] Column edit modal: clicking on right hand empty space of column name input field, should disable text selection & move cursor to the end of text; also clicking in between text should place cursor in that position. Also applicable for rename modal
- [x] Duplicate table presence alert message not displayed during table rename (it is present only for table create)

re https://github.com/nocodb/nocodb/issues/4573

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Comment by dstala** [comment]
- [x] Create a project 
    - [x] Focus on the input field
    - [x] Generate default project name
- [x] Invite user modal close on escape key press ( after inviting a user ) '
- [x] Add new row Alt R : auto focus (edit mode) on first cell in newly added row
- [x] Right align cell for numerical fields (number, decimal, currency, duration, percent). Expanded form, should be left aligned

**Comment by dstala** [comment]
issue that we saw with collaborator, is also happening with URL. so two fixes needed
- [ ] URL overflow on scroll 
- [ ] URL field- selection mode, start typing - should accept characters

**Review by dstala — APPROVED** [review]

**Review by wingkwong — COMMENTED** [review]
> Add new row Alt R : auto focus (edit mode) on first cell in newly added row
 
Just wanna double confirm that if this is expected. I create the third row, then press `ALT + R` on cell B3, should it auto focus on A3 or A4? 

image

>  Right align cell for numerical fields (number, decimal, currency, duration, percent). Expanded form, should be left aligned

It should be left aligned in expanded form.

image

File: packages/nc-gui/components/tabs/auth/user-management/UsersModal.vue [hunk 1]
```diff
@@ -149,7 +174,7 @@ const emailField = (inputEl: typeof Input) => {
     :closable="false"
     width="max(50vw, 44rem)"
     wrap-class-name="nc-modal-invite-user-and-share-base"
-    @cancel="emit('closed')"
+    @cancel="close()"
```
Comment (wingkwong) [inline packages/nc-gui/components/tabs/auth/user-management/UsersModal.vue:177]: `@cancel="close"` should work

File: packages/nc-gui/components/tabs/auth/user-management/UsersModal.vue [hunk 2]
```diff
@@ -159,7 +184,7 @@ const emailField = (inputEl: typeof Input) => {
           type="text"
           class="!rounded-md mr-1 -mt-1.5"
           data-testid="invite-user-and-share-base-modal-close-btn"
-          @click="emit('closed')"
+          @click="close()"
```
Comment (wingkwong) [inline packages/nc-gui/components/tabs/auth/user-management/UsersModal.vue:187]: `@click="close"` should work

**Comment by pranavxc** [comment]
> > Add new row Alt R : auto focus (edit mode) on first cell in newly added row
> 
> Just wanna double confirm that if this is expected. I create the third row, then press `ALT + R` on cell B3, should it auto focus on A3 or A4?
> 
> image
> 
> > Right align cell for numerical fields (number, decimal, currency, duration, percent). Expanded form, should be left aligned
> 
> It should be left aligned in expanded form.
> 
> image

1. It will focus on the first input regardless of the previous cell
2. Fixed now, good catch :+1: .

**Review by noco-w — APPROVED** [review]

**Review by wingkwong — APPROVED** [review]

**Pull request merged**